src/components/PublishedBooks/AddNewPublishedBook.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+export const AddNewPublishedBook = () => {
+  return (
+    <div className="ml-[5.5rem] mt-8 h-full border-2 border-gray-400 p-4">
+      <div className="flex flex-col  gap-5">
+        <div className="flex justify-center items-center text-2xl font-bold mb-4">
+          Add new published book:
+        </div>
+        <div>Book Title:
+        <input
+                  type="text"
+                  id="book"
+                  placeholder="Enter the book name"
+                  className="text-sm p-2 border-2 border-gray-300 rounded-md"
+                />
+        </div>
+        <div>Author name:
```

Comment:
Text shouldnt be a direct unwrapped child of a div if we can avoid it. can we use an h3 or something?

Reply:
Yeah, I still need to work on that page. I just put it there as a base for me to work on it later.